indigo/tests/bulk_creator/test_bulk_creator.py

```diff
@@ -0,0 +1,503 @@
+# -*- coding: utf-8 -*-
+import csv
+import io
+import os
+import datetime
+
+from django.test import testcases
+
+from indigo.bulk_creator import SpreadsheetRow
+from indigo.plugins import plugins
+from indigo_api.models import Country, Work
+from indigo_app.models import User
+
+
+class BulkCreateWorksTest(testcases.TestCase):
+    fixtures = ['languages_data', 'countries', 'user', 'taxonomies', 'work']
+
+    def setUp(self):
+        self.maxDiff = None
+        creator = plugins.for_locale('bulk-creator', 'za', None, None)
```

Comment:
Rather create the instance explicitly from the class, so it's clear what's being tested.

Reply:
If the ZA version is being tested, these tests should be in the tests module for indigo_za (ie. in the same project).